repository/src/main/java/org/apache/atlas/repository/store/graph/v2/tasks/ClassificationPropagationTasks.java

```diff
@@ -74,8 +74,9 @@ protected void run(Map<String, Object> parameters) throws AtlasBaseException {
                 entityGraphMapper.deleteClassificationOnlyPropagation(deletedEdgeIds);
             } else {
                 String deletedEdgeId          =  (String) parameters.get(PARAM_DELETED_EDGE_ID);
+                String classificationVertexId =  (String) parameters.get(PARAM_CLASSIFICATION_VERTEX_ID);
 
-                entityGraphMapper.deleteClassificationOnlyPropagation(deletedEdgeId);
+                entityGraphMapper.deleteClassificationOnlyPropagation(deletedEdgeId, classificationVertexId);
```

Comment:
It is tested on old tasks right?

Reply:
I do not have old task in my local setup currently. I will test that in staging

Reply:
Do one thing create task in master branch then stop Atlas in local then checkout to feature branch and then again run Atlas, so by that old task pattern will be executed in new way.

Reply:
MAke sure it works well in local first before even going into staging or beta